packages/core/src/router/router.ts

```diff
@@ -519,7 +519,7 @@ export async function performLocalNavigation(targetUrl: UrlResolvable, options?:
 			url,
 			view: {
 				component: options?.component ?? context.view.component,
-				properties: options?.properties ?? {},
+				properties: options?.properties ?? context.view.properties,
```

Comment:
Maybe checking if options.component === context.view.component the we keep the same properties and remove them otherwise